Dockerfile

```diff
@@ -1,14 +1,18 @@
 FROM nginx:1.10
 MAINTAINER Wyatt Pearsall<Wyatt.Pearsall@cfpb.gov>
 
-RUN mkdir -p /usr/src/app
+RUN if [ -z ${SKIP_JS_BUILD+x} ]; then echo "Installing JS deps" && apt-get update && \
```

Comment:
SKIP_JS_BUILD would be provided as an `arg` in hmda-platform's compose file.

However this approach might not be the right course atm due to the version of docker machine we're stuck on